detox/e2e/test/autocomplete/emoji_suggestion.e2e.ts

```diff
@@ -0,0 +1,167 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// *******************************************************************
+// - [#] indicates a test step (e.g. # Go to a screen)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element testID when selecting an element. Create one if none.
+// *******************************************************************
+
+import {Setup} from '@support/server_api';
+import {
+    serverOneUrl,
+    siteOneUrl,
+} from '@support/test_config';
+import {Autocomplete} from '@support/ui/component';
+import {
+    ChannelListScreen,
+    ChannelScreen,
+    HomeScreen,
+    LoginScreen,
+    ServerScreen,
+} from '@support/ui/screen';
+import {getRandomId, timeouts, wait} from '@support/utils';
+import {expect} from 'detox';
+
+describe('Autocomplete - Emoji Suggestion', () => {
+    const serverOneDisplayName = 'Server 1';
+    const channelsCategory = 'channels';
+    const emojiName = 'fox_face';
+    const emojiNameFirst2Chars = emojiName.substring(0, 2);
+    const emojiName3rdToLastChars = emojiName.substring(2);
+    let emojiSuggestionAutocomplete: any;
+
```

Comment:
Most of these tests are suggesting at the beginning of the input. Should we have tests that add some "lorem ipsum" at the beginning of the input and then the suggestion?

Reply:
That's covered in at_mention.e2e.ts